### ISSUE TITLE
TernError: No type found at the given position.

### ISSUE THREAD

**Issue by max-mykhailenko** [ternjs/tern_for_sublime#138]
Trying goto definition, but got this alert

**Comment by marijnh** [comment]
Without context, there is no way I'm going to be able to say anything useful about this. What code were you on when you tried to go to definition?

**Comment by max-mykhailenko** [comment]
Source file https://github.com/max-mykhailenko/react-pure-modal/blob/master/src/react-pure-modal.js
TernJS config file https://github.com/max-mykhailenko/react-pure-modal/blob/master/.tern-project

Cursor position right on `handleEsc` method
<img width="637" alt="screen shot 2016-09-23 at 10 34 37 pm" src="https://cloud.githubusercontent.com/assets/1213198/18798991/eba6ce70-81dd-11e6-93c2-d12bc4e36182.png">

**Comment by marijnh** [comment]
What did you expect to happen? That is its own definition, not an expression for which Tern can look up a definition. Try on `.close` two lines down, that should give more useful results.

**Comment by max-mykhailenko** [comment]
Plugin should just ignore pressing without alert if I'm already on definition. Maybe only with notification in status or message to console, but it's not necessary.

**Comment by marijnh** [comment]
Agreed -- that's what the Emacs plugin does. Want to write a patch to that effect?
